src/ethlance/shared/graphql/utils.cljs

```diff
@@ -0,0 +1,59 @@
+(ns ethlance.shared.graphql.utils
```

Comment:
Isn't this functionality the stuff provided by https://github.com/district0x/graphql-query/ ?

Reply:
no, the other way around - this ns is using that lib.